### PR TITLE
Support suma 50

### PR DESCRIPTION
**Pull request by mcalmer**
SUSE Manager 5.0 moved to containers and is installed by default on SLE-Micro with SELinux enabled.
This requires some changes to to storage setup as we cannot create the mountpoints everywhere and we need to keep the extendend attributes into account used for SELinux.

As the new setup tool for the server `mgradm` always create the first user, the old suma_completehook.sh can be removed.

### CODE REVIEW

**Comment by mcalmer** [comment]
I would suggest to create a 4.3 branch before merging this, as the scripts are now incompatible.

File: suma-storage-proxy [hunk 1]
```diff
@@ -53,21 +58,23 @@ create_partition $storage_disk
...
-info "Mounting storage at $storage_location"
-mount_storage $storage_disk $storage_location
+info "Mounting storage at $storage_location_tmp"
+mount_storage $storage_disk $storage_location_tmp
 
-if systemctl status squid.service >/dev/null ; then
-    squid_running=yes
-    info "Stopping squid service"
-    systemctl stop squid.service
+if systemctl status uyuni-proxy-httpd.service >/dev/null ; then
```
Comment (rjschwei) [inline suma-storage-proxy:64]: This appears to be superfluous because of the change on line 38. If the `mgrpxy` executable is found the script exits.

**Comment by rjschwei** [comment]
LGTM

**Review by rjschwei — APPROVED** [review]

**Review by jgleissner — APPROVED** [review]
LGTM.

**Pull request merged**